rasr/command.py

```diff
@@ -140,7 +179,7 @@ def run_cmd(self, cmd, args=None, retries=2):
         for t in range(retries + 1):
             try:
                 self.cleanup_before_run(cmd, t, *args)
-                sp.check_call([cmd] + [str(arg) for arg in args])
+                sp.check_call([cmd] + [str(arg) for arg in args], cwd=cwd)
```

Comment:
Just a sidenote: I think sp is imported via `from i6_core.util import *` which might be a bit hidden, maybe we should do the import manually here aswell.